src/components/collection-card/types.ts

```diff
@@ -10,3 +10,5 @@ export interface CollectionCardProps {
   url: string
   logo?: CompanyLogoOption
 }
+
+export type CollectionCardPropsWithId = CollectionCardProps & { id: string }
```

Comment:
This previously lived in `featured-in-collections/types`, now moved to `collection-card`.